graph/R-GAT/tools/accuracy_igbh.py

```diff
@@ -0,0 +1,110 @@
+import argparse
+import numpy as np
+import torch
+import json
+import os
+
+
+def get_args():
+    """Parse commandline."""
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--mlperf-accuracy-file", required=True, help="path to mlperf_log_accuracy.json"
+    )
+    parser.add_argument(
+        "--dataset-path",
+        default="igbh",
+        help="Path to IHGB dataset",
+    )
+    parser.add_argument(
+        "--dataset-size",
+        default="full",
+        choices=["tiny", "small", "medium", "large", "full"]
+    )
+    parser.add_argument(
+        "--verbose",
+        action="store_true",
+        help="verbose messages")
+    parser.add_argument(
+        "--output-file", default="coco-results.json", help="path to output file"
```

Comment:
Hi @pgmpablo157321 , would `igbh-results.json` be a more appropriate default value here?

Reply:
Yes, thanks for catching this. Updated it to `results.json`